musl/patches/handle-aux-at-base.patch

```diff
@@ -0,0 +1,48 @@
+https://git.alpinelinux.org/aports/plain/main/musl/handle-aux-at_base.patch
```

Comment:
kbuild broke?

Reply:
what?

Reply:
i meant if some of our build got broken without this patch?